src/main/java/frc/robot/subsystems/Scoring.java

```diff
@@ -26,6 +27,7 @@ public class Scoring extends PIDSubsystem {
     private final CANSparkMax rightLoadingMotor = new CANSparkMax(
             ScoringConstants.rightScoringElevatorMotorPort, MotorType.kBrushless);
     private AbsoluteEncoder leftScoringElevatorEncoder = leftScoringElevatorMotor.getAbsoluteEncoder(Type.kDutyCycle);
+    private static DigitalInput loaderLaser = new DigitalInput(ScoringConstants.loaderLaserPort);
```

Comment:
This laser should be in the storage subsystem. I recommend calling them transitionLaser and storageSensor